repository/src/main/java/org/alfresco/service/cmr/repository/ContentService.java

```diff
@@ -197,8 +223,24 @@ default DirectAccessUrl requestContentDirectUrl(NodeRef nodeRef, boolean attachm
      * @throws UnsupportedOperationException if the store is unable to provide the information.
      */
     @Auditable(parameters = {"nodeRef", "validFor"})
+    @Deprecated
     DirectAccessUrl requestContentDirectUrl(NodeRef nodeRef, boolean attachment, Long validFor);
```

Comment:
Needs a to be a default method (as mentioned above) to call the new method.

Reply:
This should be resolved now but please double check it is what you meant